tidb-cloud/set-up-private-endpoint-connections-on-google-cloud.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Connect to a TiDB Dedicated Cluster via Google Cloud Private Service Connect
+summary: Learn how to connect to your TiDB Cloud cluster via Google Cloud Private Service Connect.
+---
+
+# Connect to a TiDB Dedicated Cluster via Google Cloud Private Service Connect
+
+This document describes how to connect to your TiDB Dedicated cluster via Google Cloud Private Service Connect.
+
+> **Tip:**
+>
+> - To learn how to connect to a TiDB Dedicated cluster via private endpoint with AWS, see [Connect to TiDB Dedicated via Private Endpoint with AWS](/tidb-cloud/set-up-private-endpoint-connections.md).
+> - To learn how to connect to a TiDB Serverless cluster via private endpoint, see [Connect to TiDB Serverless via Private Endpoint](/tidb-cloud/set-up-private-endpoint-connections-serverless.md).
+
+TiDB Cloud supports highly secure and one-way access to the TiDB Cloud service hosted in a Google Cloud VPC via the [Private Service Connect](https://cloud.google.com/vpc/docs/private-service-connect). You can create an endpoint to connect to the TiDB Cloud service via the endpoint with permission.
+
+Powered by Google Cloud Private Service Connect, the endpoint connection is secure and private, and does not expose your data to the public internet. In addition, the endpoint connection supports CIDR overlap and is easier for network management.
+
+The architecture of the private endpoint is as follows:
+
+![Private Service Connect architecture](/media/tidb-cloud/google-cloud-psc-endpoint-overview.png)
+
+For more detailed definitions of the private endpoint and endpoint service, see the following Google Cloud documents:
+
+- [Private Service Connect](https://cloud.google.com/vpc/docs/private-service-connect)
+- [Access published services through endpoints](https://cloud.google.com/vpc/docs/configure-private-service-connect-services)
+
+## Restrictions
+
+- This feature is applicable to TiDB Dedicated clusters created after April 13, 2023. For older clusters, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md) for assistance.
+- Only the `Organization Owner` and `Project Owner` roles can create Google Cloud Private Service Connect endpoints.
+- You can create up to 10 endpoints for each TiDB Dedicated cluster.
```

Comment:
```suggestion
- You can create up to total 12 endpoints for each TiDB Dedicated cluster.
```

Reply:
You can create up to 10 endpoints for each Google Cloud connect to a TiDB Dedicated cluster.

Reply:
A TiDB cluster can handle connections from up to 12 endpoints.
Each Google Cloud project can have a maximum of 10 endpoints connecting to this TiDB Cluster.